controllers/inventory-controller.js

```diff
@@ -1,8 +1,11 @@
 const knex = require("knex")(require("../knexfile"));
+const { isInvIdValid, isWarehouseIdValid } = require('../utils/helpers')
 const {
   validateNewItem,
   findWarehouseIdByName,
```

Comment:
This might also be something minor, but I don't fully understand why these functions (validateNewItem, findWarehouseIdByName) were added in the require statements. Are they getting used somewhere that I don't see?

Reply:
oh nvm. I get it now.